### PR TITLE
Opsworks Provider: added custom json options

### LINKED ISSUE #271: Opsworks: override custom json
I think it's a good options to have custom_json as option in travis.

### PR DESCRIPTION
**Pull request by dieend**
Added custom_json options. If not set, will fallback to default configuration.
User have responsibility to set the custom_json value by himself. I assumed anyone who wanted to use custom_json already understand how to uses it.

closes #271

### CODE REVIEW

**Comment by jasny** [comment]
Please add the `custom_json` option to README.md.

Also you probably shouldn't edit `lib/dpl/version.rb`.

P.S. To run your own fork of use `edge: local`. See http://stackoverflow.com/questions/30584467/how-can-i-run-a-custom-version-of-dpl-on-travis-ci

**Comment by dieend** [comment]
whoops. accidentally added the version.rb in the pull request. I changed the version in reference to #146. 

fixed it. Also added the `custom_json` in readme.

File: lib/dpl/provider/ops_works.rb [hunk 1]
```diff
@@ -32,16 +32,20 @@ def check_auth
       end
 
       def custom_json
-        {
-          deploy: {
-            ops_works_app[:shortname] => {
-              migrate: !!options[:migrate],
-              scm: {
-                revision: current_sha
+        if options[:custom_json]
```
Comment (BanzaiMan) [inline lib/dpl/provider/ops_works.rb]: This `if` seems rather wordy.

How about `options[:custom_json] || { }`.

**Comment by dieend** [comment]
@BanzaiMan updated.

**Pull request merged**